includes/admin/class-wcs-admin-meta-boxes.php

```diff
@@ -353,8 +347,11 @@ public static function create_pending_parent_action_request( $subscription ) {
 	 *
 	 * @param array $available_emails
 	 * @since 1.0.0 - Migrated from WooCommerce Subscriptions v2.0
+	 *
+	 * @deprecated
 	 */
 	public static function remove_order_email_actions( $email_actions ) {
+		wcs_deprecated_function( __METHOD__, '8.3.0', 'WCS_Admin_Meta_Boxes::remove_order_email_actions' );
```

Comment:
I'm not sure if we cemented this/recorded it anywhere, but I think we recently decided that we should reference Subscriptions Prime versions for things like this, because that will make the most sense to the majority of our users:

```suggestion
		wcs_deprecated_function( __METHOD__, '7.5.0', 'WCS_Admin_Meta_Boxes::remove_order_email_actions' );
```